src/printer-postcss.js

```diff
@@ -73,17 +73,20 @@ function genericPrint(path, options, print) {
         n.value.group.type === "value-value" &&
         n.value.group.group.type === "value-func" &&
         n.value.group.group.value === "extend";
-      const isComposed =
+      const isComposedAndExternal =
         n.value.type === "value-root" &&
         n.value.group.type === "value-value" &&
-        n.prop === "composes";
+        n.prop === "composes" &&
+        n.value.group.group.groups.some(
```

Comment:
Yeah, you're right, `groups` is `undefined` when `composes` has only one class.
I added another test and updated the source.